docs/make.jl

```diff
@@ -245,7 +245,15 @@ jump_api_reference = DocumenterReference.automatic_reference_documentation(;
             "Containers.nested" => DocumenterReference.DOCTYPE_FUNCTION,
             "Containers.vectorized_product" =>
                 DocumenterReference.DOCTYPE_FUNCTION,
-            "Containers.build_ref_sets" =>
+            "Containers.build_error_fn" =>
+                DocumenterReference.DOCTYPE_FUNCTION,
+            "Containers.parse_macro_arguments" =>
+                DocumenterReference.DOCTYPE_FUNCTION,
+            "Containers.parse_ref_sets" =>
+                DocumenterReference.DOCTYPE_FUNCTION,
+            "Containers.name_with_index_expr" =>
```

Comment:
Bikeshedding: `Containers.build_name_expr`?

Reply:
I think that's a better name